testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiMinioReads.java

```diff
@@ -0,0 +1,240 @@
+/*
```

Comment:
Please add below test cases - follow up PR is also fine
* Verify the table definition using `SHOW CREATE TABLE` result
* Create a table in Hudi and drop the table in Trino

Reply:
I've added `SHOW CREATE TABLE`. I don't think drop table is supported yet. I'll check and add it as a follow up.